src/main/java/io/github/thebusybiscuit/slimefun4/utils/PatternUtils.java

```diff
@@ -20,7 +20,8 @@ private PatternUtils() {}
 
     public static final Pattern SLASH_SEPARATOR = Pattern.compile(" / ");
 
-    public static final Pattern MINECRAFT_MATERIAL = Pattern.compile("minecraft:[a-z_]+");
+    public static final Pattern MINECRAFT_NAMESPACEDKEY = Pattern.compile("minecraft:[a-z_]+");
```

Comment:
Worth noting that the namespace regex is actually `[a-z._-]+`. there are a lot of namespaces in MCs code using things like dots. I'm not sure if any are in Spigot yet but we should probably keep this regex inline with the actual regex for future-proofing
```suggestion
    public static final Pattern MINECRAFT_NAMESPACEDKEY = Pattern.compile("minecraft:[a-z._-]+");
```

Reply:
It's actually even `[a-z0-9/._-]+`, according to https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/browse/src/main/java/org/bukkit/NamespacedKey.java#35
```suggestion
    public static final Pattern MINECRAFT_NAMESPACEDKEY = Pattern.compile("minecraft:[a-z0-9/._-]+");
```

Reply:
Interesting, can't think of any with a slash off the top of my head.